Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -295,6 +295,25 @@ def audit_file
 
     problem "File should end with a newline" unless text.trailing_newline?
 
+    versioned_formulae = Dir[formula.path.to_s.gsub(/\.rb$/, "@*.rb")]
+    if !versioned_formulae.empty? && formula.tap &&
+       formula.aliases.grep(/.@\d/).empty?
+      _, last_alias_version = File.basename(versioned_formulae.sort.reverse.first)
```

Comment:
These few lines are pretty confusing with the multi-line if followed by a multi-line indented method chain. Maybe use a variable somewhere to get the `if` down to a single line?